shared/enclave/client/index.ts

```diff
@@ -0,0 +1,33 @@
+import { makeRequest } from './requests';
+import {
+  EnclaveMethods,
+  GetChainCodeParams,
+  GetChainCodeResponse,
+  SignTransactionParams,
+  SignTransactionResponse,
+  SignMessageParams,
+  SignMessageResponse,
+  DisplayAddressParams,
+  DisplayAddressResponse
+} from 'shared/enclave/types';
+
+const api = {
+  getChainCode(params: GetChainCodeParams) {
```

Comment:
We've got a lot of duplication here, could probably cut it down quite a bit with utility functions